tests/bluetooth/tester/default.conf

```diff
@@ -9,5 +9,17 @@ CONFIG_BT_ATT_PREPARE_COUNT=4
 CONFIG_BT_GATT_CLIENT=y
 CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
 CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_MESH=y
+CONFIG_BT_MESH_PROV=y
+CONFIG_BT_MESH_RELAY=n
+CONFIG_BT_MESH_PB_ADV=y
+CONFIG_BT_MESH_PB_GATT=n
+CONFIG_BT_MESH_GATT_PROXY=n
+CONFIG_BT_MESH_SUBNET_COUNT=1
+CONFIG_BT_MESH_APP_KEY_COUNT=1
+CONFIG_BT_MESH_MODEL_GROUP_COUNT=2
+CONFIG_BT_MESH_LABEL_COUNT=3
```

Comment:
Why such a limited configuration? E.g. some PTS tests require you to support at least two subnets. In fact, I'd consider enabling as many features as possible, and if there's an issue of them actually being active then we could control their enabled/disabled status at runtime. I'm thinking of things like GATT proxy here and relay here.

Reply:
You have right. I changed this in the recent version of this patch.